.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.{diff,md}]
+trim_trailing_whitespace = false
```

Comment:
We have formatting configured with prettier that runs on commit, so I don't think we want to add this, at least not the `*` rule.  Otherwise we have two sources of truth for formatting that can become in conflict with each other.  If you want to enforce formatting rules for markdown, update the `lint-staged` rule in `package.json` accordingly

Reply:
I don't want to clutter up config if you already have prettier. I will push a commit to remove this file.